services/player-service/application/src/main/java/com/kjeldsen/player/application/usecases/GeneratePlayersUseCase.java

```diff
@@ -0,0 +1,31 @@
+package com.kjeldsen.player.application.usecases;
+
+import com.kjeldsen.player.application.repositories.PlayerPositionTendencyReadRepository;
+import com.kjeldsen.player.application.repositories.PlayerWriteRepository;
+import com.kjeldsen.player.domain.Player;
+import com.kjeldsen.player.domain.PlayerPosition;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.util.stream.IntStream;
+
+@Slf4j
+@AllArgsConstructor
+@Component
+public class GeneratePlayersUseCase {
+
+    private final PlayerWriteRepository playerWriteRepository;
+    private final PlayerPositionTendencyReadRepository playerPositionTendencyReadRepository;
+
+    public void generate(int numberOfPlayers) {
+        log.info("Generating {} players", numberOfPlayers);
+        IntStream.range(0, numberOfPlayers)
+            .mapToObj(i -> playerPositionTendencyReadRepository.get(PlayerPosition.random()))
```

Comment:
why mapToObj and not just map? never seen mapToObj before

Reply:
It is the same block code that:

```
for (int i = 0; i < numberOfPlayers; i++) {
  PlayerPositionTendency randomPlayerPositionTendency = playerPositionTendencyReadRepository.get(PlayerPosition.random()));
  Player generatedPlayer = Player.generate(randomPlayerPositionTendency, 200);
  playerWriteRepository.save(player);
}
```